projects/breviary.json

```diff
@@ -1,10 +1,57 @@
 {
-  "name_cs": "Mild Blue Breviary",
-  "name_en": "Mild Blue Breviary",
+  "name_cs": "Breviary",
+  "name_en": "Breviary",
   "subtitle_cs": "Mobilní aplikace pro výpočet optimální dávky léku",
   "subtitle_en": "Medication dosage calculation app",
   "website": "https://breviary.mild.blue",
   "demo": "https://breviary.stg.mild.blue",
+  "email": "breviary@mild.blue",
+  "googlePlay": "https://breviary.mild.blue/mobile-application/android/breviary200.apk",
+  "appStore": "mailto:breviary@mild.blue",
   "desc_cs": "Breviary je jednoduchá, přehledná a rychlá mobilní aplikace pomáhající zdravotníkům snadno spočítat správné dávkování léků. Používá anonymní data a strojové učení a své výpočty tím neustále zpřesňuje. Navíc ji lze pomocí API jednoduše napojit na interní IT systémy a je tak ideálním řešením pro moderní způsob dávkování léků.",
-  "desc_en": "Virtual Breviary is an easy to use mobile app that helps healthcare workers calculate correct medication dosage in no time. Moreover, it uses anonymous data and machine learning algorithms to continuously improve the dosage computation. Ready to integrate into any hospital IT system through APIs, Mild Blue Breviary is the go to app for healthcare institutions when it comes to the right dosage calculations."
+  "desc_en": "Virtual Breviary is an easy to use mobile app that helps healthcare workers calculate correct medication dosage in no time. Moreover, it uses anonymous data and machine learning algorithms to continuously improve the dosage computation. Ready to integrate into any hospital IT system through APIs, Mild Blue Breviary is the go to app for healthcare institutions when it comes to the right dosage calculations.",
```

Comment:
Jeste ted koukam na ty texty a navrhnu upravu, deje se to tady: https://docs.google.com/document/d/1d030-5N_ajBWYVzwesSPA9PRD_78EF4OLH76aWLb_yU/edit az finalizujeme, dame vedet.